reference/docs-conceptual/samples/Working-with-Printers.md

```diff
@@ -1,27 +1,31 @@
 ---
-ms.date:  06/05/2017
+ms.date:  12/23/2019
 keywords:  powershell,cmdlet
 title:  Working with Printers
 ---
 # Working with Printers
 
-You can use Windows PowerShell to manage printers by using WMI and the WScript.Network COM object from WSH. We will use a mix of both tools to demonstrate specific tasks.
+You can use Windows PowerShell to manage printers by using WMI and the **WScript.Network** COM
+object from WSH. We will use a mix of both tools to demonstrate specific tasks.
 
 ## Listing Printer Connections
 
-The simplest way to list the printers installed on a computer is to use the WMI **Win32_Printer** class:
+The simplest way to list the printers installed on a computer is to use the WMI **Win32_Printer**
+class:
 
 ```powershell
-Get-WmiObject -Class Win32_Printer
+Get-CimInstance -Class Win32_Printer
 ```
 
-You can also list the printers by using the **WScript.Network** COM object that is typically used in WSH scripts:
+You can also list the printers by using the **WScript.Network** COM object that is typically used in
```

Comment:
This is a Windows only example.

Reply:
Changed title - the whole article is Windows-specific.